src/metaTube.cxx

```diff
@@ -1102,20 +1102,20 @@ MetaTube::M_Write()
   }
   delete[] pntVal;
 
-  unsigned int posId = M_GetPosition("id", positionUsed);
-  unsigned int posX = M_GetPosition("x", positionUsed);
-  unsigned int posY = M_GetPosition("y", positionUsed);
-  unsigned int posZ = M_GetPosition("z", positionUsed);
-  unsigned int posRed = M_GetPosition("red", positionUsed);
-  unsigned int posGreen = M_GetPosition("green", positionUsed);
-  unsigned int posBlue = M_GetPosition("blue", positionUsed);
-  unsigned int posAlpha = M_GetPosition("alpha", positionUsed);
-  unsigned int posMark = M_GetPosition("mark", positionUsed);
+  int posId = M_GetPosition("id", positionUsed);
```

Comment:
This counteracts changes in the previous commit. Rebase gone wrong?

Reply:
This is outdated.